deployments/common/crds-v1beta1/appprotect.f5.com_appolicies.yaml

```diff
@@ -480,6 +495,68 @@ spec:
                       pattern: ^http
                       type: string
                   type: object
+                grpc-profiles:
+                  items:
+                    properties:
+                      associateUrls:
+                        type: boolean
+                      attackSignaturesCheck:
+                        type: boolean
+                      defenseAttributes:
+                        properties:
+                          allowUnknownFields:
+                            type: boolean
+                          maximumDataLength:
+                            pattern: any|\d+
+                            type: string
+                        type: object
+                      description:
+                        type: string
+                      hasIdlFiles:
+                        type: boolean
+                      idlFiles:
+                        items:
+                          properties:
+                            idlFile:
+                              properties:
+                                $ref:
+                                  pattern: ^http
```

Comment:
what will happen if the file can't be downloaded from the URL? For example, the server rejects the request or times out. What is the timeout value? 

can a failure that prevent NGINX Plus from reloading?

what happens if the download is slow? let's say it takes 60s. does it mean NGINX Plus will not be able to reload until the file is downloaded?

If this can cause too many problems, perhaps we should not allow external references

Reply:
Valid points. Let's discuss offline.